docs/reference/issuers/acme/dns01/digitalocean.rst

```diff
@@ -0,0 +1,10 @@
+=========================
+DigitalOcean
+=========================
+
+.. code-block:: yaml
+
+   digitalocean:
+     tokenSecretRef:
+       name: digitalocean-dns
+       key: access-token
```

Comment:
Could you add just a few sentences to this file to explain how this should be used?

Reply:
Is it better now?

Reply:
@aslafy-z I think it may be helpful to include how to get the access key from DO (or a link to the relevant DO article). A large number of cert-manager consumers are devs are interested in getting, getting set-up and getting out and a doc like that will be super helpful. I think that's what @munnerz means here.

Reply:
Should be ok now